RecoLocalTracker/Phase2TrackerRecHits/test/RecHitsValidationTest_cfg.py

```diff
@@ -0,0 +1,68 @@
+# Imports
+import FWCore.ParameterSet.Config as cms
+
+# Create a new CMS process
+process = cms.Process('cluTest')
+
+# Import all the necessary files
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('SimGeneral.MixingModule.mixNoPU_cfi')
+process.load('Configuration.Geometry.GeometryExtended2023Dev_cff')
+process.load('Configuration.StandardSequences.MagneticField_38T_PostLS1_cff')
+process.load('Configuration.StandardSequences.RawToDigi_cff')
+process.load('Configuration.StandardSequences.L1Reco_cff')
+process.load('Configuration.StandardSequences.Reconstruction_cff')
+process.load('Configuration.StandardSequences.Validation_cff')
+process.load('DQMOffline.Configuration.DQMOfflineMC_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+process.load('Geometry.TrackerNumberingBuilder.trackerTopologyConstants_cfi')
+
+process.load('RecoLocalTracker.Phase2TrackerRecHits.Phase2TrackerRecHits_cfi')
+process.load('RecoLocalTracker.Phase2TrackerRecHits.Phase2StripCPEESProducer_cfi')
+
+# Number of events (-1 = all)
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(-1)
+)
+
+# Input file
+process.source = cms.Source('PoolSource',
+    fileNames = cms.untracked.vstring('file:step3.root')
+)
+
+# TAG
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:upgradePLS3', '')
+
+# Output
+process.TFileService = cms.Service('TFileService',
+    fileName = cms.string('file:rechits_validation.root')
+)
+
+# DEBUG
+process.MessageLogger = cms.Service('MessageLogger',
+	debugModules = cms.untracked.vstring('siPhase2RecHits'),
+	destinations = cms.untracked.vstring('cout'),
+	cout = cms.untracked.PSet(
+		threshold = cms.untracked.string('ERROR')
+	)
+)
+
+
+# Analyzer
+process.analysis = cms.EDAnalyzer('Phase2TrackerRecHitsValidation',
+    src = cms.InputTag("siPhase2RecHits")
+)
+
+# Processes to run
+process.rechits_step = cms.Path(process.siPhase2RecHits)
+process.validation_step = cms.Path(process.analysis)
+
+process.schedule = cms.Schedule(process.rechits_step, process.validation_step)
+
+from SLHCUpgradeSimulations.Configuration.phase2TkCustomsBE5DPixel10Ddev import customise_condOverRides
```

Comment:
Sure it would be nice, but we don't even have the digis yet. We're not at that stage yet, and this can be cleaned up when we reach there.